eox_nelp/external_certificates/models.py

```diff
@@ -0,0 +1,102 @@
+"""
+Models file for the external_certificates.
+"""
+import logging
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils import timezone
+
+User = get_user_model()
+logger = logging.getLogger(__name__)
+
+
+class ExternalCertificate(models.Model):
+    """
+    Model to store information about external certificates.
+
+    Attributes:
+        user (User): The user associated with the certificate.
+        group_code (str): Group code associated to the cert.
+        certificate_id (str): Unique identifier of the certificate.
+        course_id (str): Group code used to generate the certificate.
+        certificate_url_en (str): URL to access the certificate in English.
+        certificate_url_ar (str): URL to access the certificate in Arabic.
+        created_at (datetime): Timestamp when the certificate was created.
+    """
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="external_certificates",
+        help_text="The user associated with this certificate."
+    )
+    group_code = models.CharField(
```

Comment:
How are you getting this value ?

Reply:
Changed to [8261b9a](https://github.com/eduNEXT/eox-nelp/pull/266/commits/8261b9a5ac825f1f2c930cee2ca79552d99f746f)